### PR TITLE
Impacthub-3 config setup guide and new genesis

### PR DESCRIPTION
**Pull request by franono**

### CODE REVIEW

File: impacthub-3/README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # ixo Network Validator for Impacthub-3
 
 - If you're upgrading from impacthub-2 [click here](./README_UPGRADE.md)
-- If you're installing from scratch (**Coming Soon**) <!--[click here](./README_INSTALL.md)-->
+- If you're installing from scratch [click here](./README_INSTALL.md)
+- If you're looking for validator setup guide [click here](./README_UPGRADE.md)
```
Comment (migueldingli1997) [inline impacthub-3/README.md:5]: Should this be `README_CONFIG`?
```suggestion
- If you're looking for validator setup guide [click here](./README_CONFIG.md)
```

File: impacthub-3/README_CONFIG.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+```
+{
+  "latest_block_hash": "6B87878277C9164006F2E7C544A27C2A4010D0107F436645BFE35BAEBE50CDF2",
+  "latest_app_hash": "010EF4A62021F88D097591D6A31906CF9E5FA4359DC523B535E3C411DC6010B1",
+  "latest_block_height": "233053",
+  "latest_block_time": "2020-01-31T22:12:45.006715122Z",
+  "catching_up": true
+}
+```
+The main thing to watch is that the block height is increasing. Once you are caught up with the chain, catching_up will become false. At that point, you can start using your node to create a validator.
```
Comment (migueldingli1997) [inline impacthub-3/README_CONFIG.md:99]:
```suggestion
The main thing to watch is that the block height is increasing. Once you are caught up with the chain, `catching_up` will become false. At that point, you can start using your node to create a validator.
```

**Comment by franono** [comment]
yes ty

File: impacthub-3/README_CONFIG.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+  "latest_block_height": "233053",
+  "latest_block_time": "2020-01-31T22:12:45.006715122Z",
+  "catching_up": true
+}
+```
+The main thing to watch is that the block height is increasing. Once you are caught up with the chain, `catching_up` will become false. At that point, you can start using your node to create a validator.
+
+## Registering validator
+
+If you are upgrading, view [README_UPGRADE.md](./README_INSTALL.md).
```
Comment (migueldingli1997) [inline impacthub-3/README_CONFIG.md:103]:
```suggestion
If you are upgrading, view [README_UPGRADE.md](./README_UPGRADE.md).
```

File: pandora-4/README_CONFIG.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+  "latest_block_height": "233053",
+  "latest_block_time": "2020-01-31T22:12:45.006715122Z",
+  "catching_up": true
+}
+```
+The main thing to watch is that the block height is increasing. Once you are caught up with the chain, `catching_up` will become false. At that point, you can start using your node to create a validator.
+
+## Registering validator
+
+If you are upgrading, view [README_UPGRADE.md](./README_INSTALL.md).
```
Comment (migueldingli1997) [inline pandora-4/README_CONFIG.md:103]:
```suggestion
If you are upgrading, view [README_UPGRADE.md](./README_UPGRADE.md).
```

**Pull request merged**